examples/apps/collaborative-textarea/package.json

```diff
@@ -37,6 +37,7 @@
     "@fluidframework/core-interfaces": "^0.30.0",
     "@fluidframework/get-tinylicious-container": "^0.30.0",
     "@fluidframework/react-inputs": "^0.30.0",
+    "@fluidframework/runtime-utils": "^0.30.0",
```

Comment:
i'm not sure we want to have such and explicitly dependency on the runtime from a data store. we kind of already do that via aqueduct, but i think that is wrong as well.

Reply:
runtime-utils only has one really nasty dependency: @fluidframework/protocol-base

---
In reply to: [529902723](https://github.com/microsoft/FluidFramework/pull/4246#discussion_r529902723) [](ancestors = 529902723)